node/src/p2p.rs

```diff
@@ -327,6 +330,10 @@ impl Worker {
             SwarmEvent::ConnectionClosed { peer_id, .. } => {
                 self.peer_tracker.remove(peer_id);
             }
+            #[cfg(debug_assertions)]
+            SwarmEvent::NewListenAddr { address, .. } => {
+                info!("listening on: {address}");
+            }
```

Comment:
why the `#[cfg(debug_assertions)]`?

Reply:
to lessen the impact :P I added that for testing connection between nodes manually, I'll remove that now